tests/json_queries.cc

```diff
@@ -366,3 +403,346 @@ TEST(QueryHandler, AddAndFind)
     VDMSConfig::destroy();
     PMGDQueryHandler::destroy();
 }
+
+TEST(BoundingBox, simpleAdd)
```

Comment:
most of these testing is repeated in python. We shoudl just stick to python for this testing and leaving the C++ testing when checking stuff related to PMGDQueryHandler and things other than right responses. 
If we want to add checking of query-responses in C++, we should use the C++ client library. 
Instead of doing that, better to move these tests to python, most of which you already put in python.